hack/install.sh

```diff
@@ -129,19 +129,19 @@ install_faasd() {
 }
 
 install_caddy() {
-  if [ ! -z "${FAASD_DOMAIN}" ]; then
+  if [ -n "${FAASD_DOMAIN}" ]; then
     CADDY_VER=v2.4.3
     arkade get --progress=false caddy -v ${CADDY_VER}
-    
+
     # /usr/bin/caddy is specified in the upstream service file.
     $SUDO install -m 755 $HOME/.arkade/bin/caddy /usr/bin/caddy
 
     $SUDO curl -fSLs https://raw.githubusercontent.com/caddyserver/dist/master/init/caddy.service --output /etc/systemd/system/caddy.service
 
     $SUDO mkdir -p /etc/caddy
     $SUDO mkdir -p /var/lib/caddy
-    
-    if $(id caddy >/dev/null 2>&1); then
+
+    if id caddy >/dev/null 2>&1; then
```

Comment:
This syntax does not look right?

Reply:
I agree is looks funky, but removing the parens fixes the SC2091 warning in `shellcheck` and it still evaluates the same.

Reply:
I am not a shellcheck user so this doesn't bother me. I prefer the clarity we had originally.